### ISSUE TITLE
Intellj Idea 2020.2 lombok bug

### ISSUE THREAD

**Issue by wellhor** [mplushnikov/lombok-intellij-plugin#842]
when I use @RequiredArgsConstructor  in a class, idea report a error.

like this
```
@RequiredArgsConstructor
public class XXCache implements ICache {

    private final FlowService flowService;

}
```
The field report a error: Variable 'flowService' might not have been initialized

**Comment by alex-ureche** [comment]
Plugin incompatibility with 2020.2 is not yet fixed.
https://github.com/mplushnikov/lombok-intellij-plugin/issues/840

**Comment by elbraulio** [comment]
this https://github.com/mplushnikov/lombok-intellij-plugin/issues/840#issuecomment-665115639 was the solution for me.

**Comment by uinio** [comment]
delete 
 C:\Users\user\AppData\Local\JetBrains\IntelliJIdea2020.1
 C:\Users\user\AppData\Roaming\JetBrains\IntelliJIdea2020.1

**Comment by hrefr** [comment]
> this [#840 (comment)](https://github.com/mplushnikov/lombok-intellij-plugin/issues/840#issuecomment-665115639) was the solution for me.

Didn't work for me, unfortunately. Uninstalled plugin, restarted IDEA, installed plugin, restarted IDEA. Still have to restart IDEA every time I add new private final fields to a class marked with @RequiredArgsConstructor.

**Comment by elbraulio** [comment]
seems to be another issue that is being solved here #860. Unfortunately, the owner is on his vacations and we have to wait until he fixes this issue. Maybe you can downgrade your Intellij version. I don't know if that may fix the issue.

**Comment by hrefr** [comment]
> seems to be another issue that is being solved here #860. Unfortunately, the owner is on vacation and we have to wait until he fixes this issue. Maybe you can downgrade your Intellij version. I don't know if that may fix the issue.

Oh, yeah, I'll wait. It's sure annoying but bearable for now. Thanks for pointing master issue. I just noticed those "psi is outdated" exceptions while playing with @RequiredArgsConstructor annotation, so it's definitely related.

**Comment by wojciechmusial** [comment]
Lombok should be natively supported by JetBrains, could you guys comment, upvote this: https://youtrack.jetbrains.com/issue/IDEABKL-5667? Maybe they will finally do something about it, with all due respect for the author, millions of devs shouldn't be dependent on single maintainer plugin.
